Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollJointLimitWidget.h

```diff
@@ -23,6 +24,25 @@ QT_FORWARD_DECLARE_CLASS(QLabel)
 namespace EMotionFX
 {
     class ObjectEditor;
+    class RagdollJointLimitWidget;
+
+    class RagdollJointLimitPropertyNotify
+        : public AzToolsFramework::IPropertyEditorNotify
+    {
+    public:
+        RagdollJointLimitPropertyNotify(RagdollJointLimitWidget* ragdollJointLimitWidget);
+
+        void BeforePropertyModified([[maybe_unused]] AzToolsFramework::InstanceDataNode* node) override {};
+        void AfterPropertyModified(AzToolsFramework::InstanceDataNode* node) override;
+
+        void SetPropertyEditingActive([[maybe_unused]] AzToolsFramework::InstanceDataNode* node) override {}
+        void SetPropertyEditingComplete([[maybe_unused]] AzToolsFramework::InstanceDataNode* node) override {};
+
+        void SealUndoStack() override {}
```

Comment:
uber nit: these override could be together under the comment `// AzToolsFramework::IPropertyEditorNotify overrides ...`

Reply:
Added